VMcollector: update documentation (#4719)
kyma-project/test-infra: development/tools/cmd/vmscollector/README.md

## Overview

This command finds and removes orphaned virtual machines (VMs) created by Prow jobs in a Google Cloud Platform (GCP) project.

> Prow jobs creates a VM instance to install and test Kyma.

Reviewer: ```suggestion
Prow jobs create a VM instance to install and test Kyma.
```

Usually, the job also cleans up the instance.
It can happen, however, that the job is terminated before its clean-up finishes.
This causes a resource leak that generates unwanted costs.
The garbage collector finds and removes such VM instances.

There are three conditions used to find instances for removal:

> - The instance name is not caught by the exclud names regexp

Reviewer: ```suggestion
- The instance name is not caught by the exclude names regexp
```

> - The value of a `job-name` label the instance is annotated with is not caught by the exclud labels regexp

Reviewer: ```suggestion
- The value of the `job-name` label the instance is annotated with is not caught by the exclude labels regexp
```

> - The instance `creationTimestamp` value that is used to find instance existing at least for a preconfigured number of hours

Reviewer: ```suggestion
- The instance `creationTimestamp` value that is used to find instance existing at least for a preconfigured number of hours.
```

## Overview

This command finds and removes orphaned virtual machines (VMs) created by Prow jobs in a Google Cloud Platform (GCP) project.

Prow jobs create a VM instance to install and test Kyma.
Usually, the job also cleans up the instance.
It can happen, however, that the job is terminated before its clean-up finishes.
This causes a resource leak that generates unwanted costs.
The garbage collector finds and removes such VM instances.

There are three conditions used to find instances for removal:

> - The instance name is not caught by the exclude names regexp

Reviewer: ```suggestion
- The instance name is not caught by the exclude names regexp.
```

> - The instance name is not caught by the exclude names regexp.

Reviewer: ```suggestion
- The instance name is not caught by the exclude names regex.
```

Reviewer: According to Google, it's `regex`, not `regexp` 🤔 
Could you verify and change it in the rest of the document if needed?

Author: Done

> - The value of the `job-name` label the instance is annotated with is not caught by the exclude labels regexp

Reviewer: ```suggestion
- The value of the `job-name` label the instance is annotated with is not caught by the exclude labels regexp.
```
